ansible/vars/default.yaml

```diff
@@ -42,6 +45,9 @@ ocp_worker_disk: 120
 # OCP cluster name
 ocp_cluster_name: ostest
 
+# Kubeconfig default path
+kubeconfig_path: "{{ base_path }}/dev-scripts/ocp/{{ ocp_cluster_name }}/auth/kubeconfig"
```

Comment:
Presumably this is to support initialise local_oc_client using an installer other than dev-scripts?

Reply:
This is gone now anyhow.